desktop-app/app/components/WebView/index.js

```diff
@@ -449,51 +456,6 @@ class WebView extends Component {
           responsivelyApp.domInspector.enable();
         }
       });
-
-      document.addEventListener('scroll', (e) => {
-        if (!responsivelyApp.mouseOn) {
-          return;
-        }
-        window.responsivelyApp.sendMessageToHost(
-          '${MESSAGE_TYPES.scroll}',
-          {
-            position: {x: window.scrollX, y: window.scrollY},
-          }
-        );
-      });
-
-      document.addEventListener(
-        'click',
-        (e) => {
-          if (e.target === window.responsivelyApp.lastClickElement || e.responsivelyAppProcessed) {
-            window.responsivelyApp.lastClickElement = null;
-            e.responsivelyAppProcessed = true;
-            return;
-          }
-          if (window.responsivelyApp.domInspectorEnabled) {
```

Comment:
@esprush Removing this will break the inspector, I guess.

Reply:
changed @manojVivek . removed only the section where click event was sent.